lib/samson/secrets/vault_client_manager.rb

```diff
@@ -103,7 +103,7 @@ def client(deploy_group_permalink)
       private
 
       def with_retries(&block)
-        Vault.with_retries(Vault::HTTPConnectionError, attempts: 3, &block)
+        Vault.with_retries(Vault::HTTPConnectionError, attempts: 5, wait_time: 0.1, &block)
```

Comment:
this is using this gem? https://github.com/zendesk/vault-ruby#making-requests
`wait_time` isn't mentioned in the docs

Reply:
code is here https://github.com/hashicorp/vault-ruby/blob/v0.12.0/lib/vault/client.rb#L412

Reply:
and it comes with a builtin backoff ... so yeah this needs to either just bump retries or change `base` option